PersonStageThree/src/com/beyond/person/core/service/implementation/DancerImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.beyond.person.core.service.implementation;
+
+import com.beyond.person.core.exception.InvalidFieldException;
+import com.beyond.person.core.model.Dancer;
+import com.beyond.person.core.service.interfaces.DancerAction;
+import com.beyond.person.core.utils.State;
+
+public class DancerImpl implements DancerAction {
```

Comment:
@maria-salmasian name should be "DancerActionServiceImpl" and interface should be DancerActionService applicable for all types

Reply:
changed it